packages/app/public/config.js

```diff
@@ -1,4 +1,4 @@
-window.KEYCLOAK_URL = 'http://localhost:8093/auth';
+window.KEYCLOAK_URL = 'http://localhost:8082/auth';
```

Comment:
Hmmm what is the difference between keycloak on 8093 and 8082? Locally we have both but I'm wondering why this change is needed

Reply:
According to the [README of the backend-libraries](https://github.com/nl-portal/nl-portal-backend-libraries#getting-started), we need to run the docker compose in [this repo](https://github.com/nl-portal/nl-portal-docker-compose). Running this, will start the "nl-portal-keycloak" service [on port 8082](https://github.com/nl-portal/nl-portal-docker-compose/blob/main/docker-compose.yaml#L10).